README.md

```diff
@@ -248,6 +248,18 @@ As an example we use the Symfony `DateTimeNormalizer` service so we do have supp
 
 As a side note: If you happen to use [Autowiring](https://symfony.com/doc/current/service_container/autowiring.html) in your `services.yaml` you might need to set `autowire: false` too.
 
+**When namespace of Model used change**
+
+You must have migrated the data in the database because your JSON has `#type` key which contains the namespace of your used Model class.
```

Comment:
```suggestion
Because we store the `#type` along with the data in the database, you have to migrate the already existing data in your database to reflect the new namespace.
```